docs/source/tutorials/indepth_overview.ipynb

```diff
@@ -77,6 +77,24 @@
     "%config InlineBackend.print_figure_kwargs={\"facecolor\": \"w\"}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%HTML\n",
```

Comment:
Won't this be visible in the docs build and in the colab notebook?

Reply:
At least need to replace `metadata` above with:

```"metadata": {
    "nbsphinx": "hidden"
   },
```

That will at least hide in the docs website.

Reply:
Yes I believe so, the code will be visible. I'm not sure how to hide it though.

Reply:
Actually if you want to add this stuff to this notebook it should go into the cell that is already hidden (the one starting with `# Package installation.`)

Reply:
Sounds good, agree. I moved it to the cell that is already hidden.